### ISSUE TITLE
PILA Thailand - teacher account creation adjustment to ensure they can reload

### ISSUE THREAD

**Issue by OECDPILA** [pila-project/pila-platform#148]
(Medium priority)

Once a teacher creates an account, they see the message about sending their ID and requesting access. Stephane rightly pointed out that it would be good to have a 'reload' or refresh button that says something like "Once you receive confirmation that your request is approved, click on the XXXX button". 

I will ask the translator to do a last sweep on translations in the PILA account on Tuesday to capture these small changes. Thanks!

**Comment by OECDPILA** [comment]
low priority for now
